.env.sample

```diff
@@ -12,3 +12,6 @@ export FACEBOOK_KEY=<APP ID>
 export FACEBOOK_SECRET=<App Secret>
 export GITHUB_KEY=<Client ID>
 export GITHUB_SECRET=<Client Secret>
+
+# Heroku Credentials
+export HEROKU_BACKEND_URL=http://127.0.0.1:8000
```

Comment:
I wanted to make sure all my teammates update their .env files. But that is not the real url, it only works in localhost.

Reply:
Because in the models I removed the hardcoded urls and moved the domain to .env so that our backend url is not exposed. So I was just updating the .env-example so that my teammates update theirs too.